_sub/compute/helm-crossplane/versions.tf

```diff
@@ -1,3 +1,10 @@
 terraform {
   required_version = ">= 0.12"
+
+  required_providers {
+
+    kubectl = {
+      source  = "gavinbunney/kubectl"

```

Comment:
This could be a potential RAT carrier. But as such I recon you guys trust Gavin :)

Reply:
Yeah I thought the same but we're using it in here already and all seems legitimate

Reply:
@ZaradarDFDS More like RAbbiT!